crates/release_plz/src/args/release.rs

```diff
@@ -43,18 +44,33 @@ pub struct Release {
     /// Git token used to publish the GitHub release.
     #[clap(long, value_parser = NonEmptyStringValueParser::new())]
     pub git_token: Option<String>,
+    /// Kind of git backend
+    #[arg(long, value_enum, default_value_t = GitBackendKind::Github)]
+    backend: GitBackendKind,
 }
 
 impl TryFrom<Release> for ReleaseRequest {
     type Error = anyhow::Error;
 
     fn try_from(r: Release) -> Result<Self, Self::Error> {
+        let git_token = SecretString::from(
+            r.clone()
+                .git_token
+                .context("git_token is required for git_release")?,
+        );
         let git_release = if r.git_release {
             let release = release_plz_core::GitRelease {
-                git_token: SecretString::from(
-                    r.git_token
-                        .context("git_token is required for git_release")?,
-                ),
+                git_token: git_token.clone(),
+                backend: match r.backend {
+                    GitBackendKind::Gitea => {
+                        GitBackend::Gitea(Gitea::new(r.repo_url()?, git_token)?)
```

Comment:
instead of typing `r.repo_url()?` many times, please save it in a variable before.
```rust
let repo_url = r.repo_url()?;
```